doc/BUILD-Debian.md

```diff
@@ -0,0 +1,74 @@
+1. install boost (apt-get)
+2. install NVidia drivers
+3. install CUDA
+4. install ccmake
```

Comment:
Is it a really good idea to install ccmake from a package (step 4) and install CMake from source afterwards (step 10)?

Reply:
yes good point, we should remove (4).

On the other hand, we should also decide the purpose of this file, whether it is to remind us how to build it on a particular machine (Debian 8) or more a general guide for a Debian machine. Because I guess things may have evolved since 2 years ago (Debian 9).

Reply:
Yes, we could also remove this file and update the main `BUILD.md`.